galley/pkg/config/analysis/msg/generate.main.go

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:build ignore
```

Comment:
Are they both needed?

Reply:
Yes, gofmt and govet will require them both starting in 1.17

Reply:
ACK